SeaPublicWebsite.Data/ErrorHandling/CustomErrorPageExceptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace SeaPublicWebsite.Data.ErrorHandling
```

Comment:
Why is this in the data project? The data project should know nothing about the website